core/schema/container.go

```diff
@@ -489,6 +492,10 @@ func (s *containerSchema) Install() {
 			Doc(`The error stream of the last executed command.`,
 				`Will execute default command if none is set, or error if there's no default.`),
 
+		dagql.Func("exitCode", s.exitCode).
+			Doc(`The exit code of the last executed command.`,
+				`Will execute default command if none is set, or error if there's no default.`),
```

Comment:
The "execute default command" part was only true before v0.12.0. Since this is a new field, it won't even show in that version.

Reply:
Hmmm, I notice `stdout` and `stderr` also have it for `AllVersion`. 🤔

Reply:
Seems like an oversight in https://github.com/dagger/dagger/pull/7857/files#diff-89b0b6c38f29bb6615327b6126c2479d29b032ef974017a9358c0c0721fa17f9

Reply:
Should say:
```diff
-				`Will execute default command if none is set, or error if there's no default.`),
+				`Returns an error if no command was set.`),
```

Reply:
Ah, sorry, github didn't refresh the last message :cry: looks like we realized the same thing.

Will update all the messages.